crates/core_arch/src/x86/sse.rs

```diff
@@ -1401,6 +1410,16 @@ pub unsafe fn _mm_getcsr() -> u32 {
 /// * The *denormals-are-zero mode flag* turns all numbers which would be
 /// denormalized (exponent bits are all zeros) into zeros.
 ///
+/// Note that modfying the masking flags, rounding mode, or denormals-are-zero mode flags leads to
+/// **immediate Undefined Behavior**: Rust assumes that these are always in their default state and
```

Comment:
This makes this function basically impossible to use in a UB-free way. Should we deprecate it in favor of inline assembly blocks?

Reply:
Yes